.gitlab-ci.yml

```diff
@@ -30,21 +30,23 @@ variables:
 
 before_script:
   - ./tests/scripts/rebase.sh
-  - /usr/bin/python -m pip install -r tests/requirements.txt
+  - update-alternatives --install /usr/bin/python python /usr/bin/python3 1
+  - python -m pip install -r tests/requirements.txt
   - mkdir -p /.ssh
 
 .job: &job
   tags:
     - packet
   variables:
-    KUBESPRAY_VERSION: v2.10.0
+    KUBESPRAY_VERSION: v2.11.0
   image: quay.io/kubespray/kubespray:$KUBESPRAY_VERSION
 
 .testcases: &testcases
   <<: *job
   services:
     - docker:dind
   before_script:
+    - update-alternatives --install /usr/bin/python python /usr/bin/python3 1
```

Comment:
why two times?

Reply:
@mattymo jobs that redefine the `before_script` (like `.testcases` and `.terraform_install` will not get the `update-alternatives` from line 33